### PR TITLE
create,update, delete user

### PR DESCRIPTION
**Pull request by tuanbpl-0873**
https://edu-redmine.sun-asterisk.vn/issues/10628

### CODE REVIEW

**Comment by tuanbpl-0873** [comment]
ready

File: resources/views/admin/layouts/master.blade.php [hunk 1]
```diff
@@ -149,8 +149,33 @@ class="fa fa-search"></i>
...
+    <script src="{{ asset('js/admin/master.js') }}"></script>
     <script src="{{ asset('bower_components/toastr/toastr.js') }}"></script>
+
+    <script>
+        @if(Session::has('message'))
+        var type = "{{ Session::get('alert-type', 'info') }}";
+
+        switch(type){
+            case 'info':
+                toastr.info("{{ Session::get('message') }}","Infor Alert", {timeOut: 8000});
```
Comment (thucpv-1748) [inline resources/views/admin/layouts/master.blade.php]: convertion

File: resources/views/admin/layouts/master.blade.php [hunk 2]
```diff
@@ -149,8 +149,33 @@ class="fa fa-search"></i>
...
 
-    <script src="{{ asset('js/admin/custom.js') }}"></script>
+    <script src="{{ asset('js/admin/master.js') }}"></script>
     <script src="{{ asset('bower_components/toastr/toastr.js') }}"></script>
+
+    <script>
+        @if(Session::has('message'))
+        var type = "{{ Session::get('alert-type', 'info') }}";
+
+        switch(type){
```
Comment (thucpv-1748) [inline resources/views/admin/layouts/master.blade.php]: move vao file js

File: resources/views/admin/users/update.blade.php [hunk 3]
```diff
@@ -0,0 +1,153 @@
+@extends('admin.layouts.master')
+
+@section('title', 'Update Users')
```
Comment (thucpv-1748) [inline resources/views/admin/users/update.blade.php]: I18n

File: resources/views/admin/users/update.blade.php [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+                                @endforeach
+                            </ul>
+                        </div>
+                        @endif
+                        <form action="{{ route('users.update', $user->id) }}" method="post">
+                            @method('PUT')
+                            {{ csrf_field() }}
+                            <div class="form-group">
+                                <label for="name">{{ __('Name') }}</label>
+                                <input type="text" class="form-control" name="name" value="{{ $user->name}}">
```
Comment (thucpv-1748) [inline resources/views/admin/users/update.blade.php]: convertion

File: resources/views/admin/users/update.blade.php [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+                                <input type="email" class="form-control" name="email" value="{{ $user->email}}">
+                            </div>
+                            <div class="form-group">
+                                <label for="phone">{{ __('Phone') }}</label>
+                                <input type="text" class="form-control" name="phone" value="{{ $user->phone}}">
+                            </div>
+                            <div class="form-group">
+                                <label for="gender">{{ __('Gender') }}</label>
+                                <select class="form-control" name="gender">
+                                    <option value='female' @if ($user->gender == 'female') selected @endif>{{ __('Female') }}</option>
```
Comment (thucpv-1748) [inline resources/views/admin/users/update.blade.php]: loạn phẩn đơn kép

File: resources/views/admin/users/update.blade.php [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+    </section>
+    <!-- /.content -->
+</div>
+<!-- /.content-wrapper -->
+@endsection
+
+@section('script')
+
+@parent
+@endsection
```
Comment (thucpv-1748) [inline resources/views/admin/users/update.blade.php]: thieu dong trắng

File: resources/views/admin/layouts/master.blade.php [hunk 7]
```diff
@@ -149,8 +149,33 @@ class="fa fa-search"></i>
     <!-- AdminLTE App -->
     <script src="{{ asset('bower_components/admin-lte/dist/js/adminlte.min.js') }}"></script>
 
-    <script src="{{ asset('js/admin/custom.js') }}"></script>
+    <script src="{{ asset('js/admin/master.js') }}"></script>
     <script src="{{ asset('bower_components/toastr/toastr.js') }}"></script>
+
+    <script>
```
Comment (BunnyPi04) [inline resources/views/admin/layouts/master.blade.php]: viết riêng ra

**Comment by tuanbpl-0873** [comment]
ready

**Comment by tuanbpl-0873** [comment]
ready

File: app/Http/Controllers/Admin/UserController.php [hunk 8]
```diff
@@ -46,7 +47,33 @@ public function create()
...
+            'phone' => $request->phone,
+            'address' => $request->address,
+            'permission' => $request->permission,
+            'gender' => $request->gender,
+            'password' => Hash::make($inputPassword),
+        ];
+
+        $this->user->create($userInfor);
+        
+        $notification = array(
```
Comment (BunnyPi04) [inline app/Http/Controllers/Admin/UserController.php]: https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_27-mang-va-chuoi-12
2.7.2 nhé
sử dụng [] thay vì array()

File: resources/views/admin/layouts/master.blade.php [hunk 9]
```diff
@@ -138,6 +138,8 @@ class="fa fa-search"></i>
         <footer class="main-footer">
             <!-- Default to the left -->
             <strong>{{ __('Copyright ') }}&copy; {{ __('2016') }} <a href="#">{{ __('Sun') }}</a>.</strong> {{ __('All rights reserved.') }}
+            <input type="hidden" class="toastr-session"
+            data-session="{{ Session::has('message') }},{{ Session::get('alert-type', 'info') }},{{ Session::get('message') }}" >
```
Comment (BunnyPi04) [inline resources/views/admin/layouts/master.blade.php]: convention

**Comment by tuanbpl-0873** [comment]
ready

**Pull request merged**